### PR TITLE
made py-button styling dynamic

### PR DESCRIPTION
**Pull request by Bruno-Odinukweze**
## What?
I've added a functionality to enable dynamic styling of  `py-button`
## Why?
These changes makes it very easy for the user to define their own style, or give the button a style of their choice
## How?
This includes a `styling` attribute (also dynamic) inside the `py-button`  tag. The Styling attribute works the same way as a `class` in a HTML tag
## Testing?
I've also tested and make sure it works alongside everything else

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,9 @@ __pycache__/
 *$py.class
 .DS_Store
 
+# TODO File
+my_contributions.txt
+
```
Comment (fpliger) [inline .gitignore:9]: Can you remove this? Seem pretty related to your personal workflow (trying to keep entropy under control)

**Review by fpliger — CHANGES_REQUESTED** [review]
Awesome, thank you @Bruno-Odinukweze for the contribution! Left some minor comments that would be good to be addressed before we merged but the PR lgtm overall! 👍

File: .gitignore (hunk 1 above)
Comment (Bruno-Odinukweze) [reply .gitignore]: Thanks @fpliger. I've made the requested changes

**Comment by Bruno-Odinukweze** [comment]
Thanks for the feedback @fpliger, I've made the requested changes

**Comment by fpliger** [comment]
Will merge because this is a good improvement. TY @Bruno-Odinukweze ! Just for the sake of information, this may become obsolete or change soon as we support theming but is a great start :)

**Comment by fpliger** [comment]
Lint is failing.... I'm pretty sure all you need is to replace `let` with `const` on a couple of the new variables you introduced with this PR. 

Thank you!

**Comment by Bruno-Odinukweze** [comment]
Hi @fpliger, glad to know about the theming, looking forward to contributing more :)

**Review by fpliger — APPROVED** [review]

**Pull request merged**